lib/types/uuid.js

```diff
@@ -18,6 +18,10 @@ const uuidRegex =
  * Represents an immutable universally unique identifier (UUID). A UUID represents a 128-bit value.
  */
 class Uuid {
+    /**
+     * @type {rust.UuidWrapper}
+     * @private
+     */
```

Comment:
Add proper description for people developing this code in the future

Reply:
I do not think a comment would add a lot more decription that what can be extracted from JSDocs for this functon.